WatchFaceAlphaKotlin/app/build.gradle

```diff
@@ -66,13 +66,13 @@ dependencies {
     implementation 'androidx.wear:wear:1.1.0'
 
     // Watch face specific libraries
-    implementation "androidx.wear:wear-complications-data:$rootProject.watchFaceAlphaVersion"
-    implementation "androidx.wear:wear-complications-provider:$rootProject.watchFaceAlphaVersion"
+    implementation "androidx.wear:wear-complications-data:$rootProject.watchFaceVersion"
+    implementation "androidx.wear:wear-complications-provider:$rootProject.watchFaceProviderVersion"
```

Comment:
`wear-complications-provider` was replaced by `wear-complications-data-source`, so we should update that here.

I'm actually slightly surprised this works as is, I wonder if `wear-watchface-complications-rendering` transitively exposes `wear-complications-data-source`? And then no classes conflict?

Reply:
I was using alpha 17 where it didn't exist but, yeah, not sure how I got it.  :D

Oh well, I bumped to 19 and renamed it.